app/controllers/global_search_controller.rb

```diff
@@ -14,9 +12,14 @@ def self.searcher_classes
   end
 
   def index
-    @searchers = self.class.searcher_classes.map do |searcher_class|
-      searcher_class.new(current_user, current_facility, params[:search])
-    end
+    @searchers = if session_user.nil? || session_user != acting_user
```

Comment:
Looks like we have a `acting_as?` on ApplicationController that does something similar. Could we use that? I think Devise gives us a `logged_in?` helper. Would `if logged_in? && !acting_as?` be more intention-revealing? (I think what I said requires would flip the conditional)